src/components/organisms/NewProfileModal/components/header/ProfileModalBasicTextInfo/ProfilModalBasicTextInfo.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../profileModalStyles";
+
+$name-bottom-margin: 4px;
+$light-color: #d2d2d2;
```

Comment:
move to main consts

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/53d93fd0cceb2fc3b03502c6df2e213fb9dce7be